owasp-security-logging-log4j/pom.xml

```diff
@@ -31,8 +31,19 @@
     <url>https://github.com/javabeanz/owasp-security-logging</url>
   </scm>
   <properties>
-    <log4j.version>2.16.0</log4j.version>
+    <log4j2.version>2.16.0</log4j2.version>
   </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-bom</artifactId>
```

Comment:
If we import log4j-bom do we still need log4j-core, log4j-api, and log4j-slf4j-impl?

Reply:
yes, the way a BOM works is that expose those for "use" in the regular dependencies section but doesn't pull them in until they are directly reference below.  The nice thing is that you can then reference them sans the version tag in the POM below.